### ISSUE TITLE
ComfyUI getting the wrong positive prompt when loading workflow from image

### ISSUE THREAD

**Issue by brunodpoliveira** [comfyanonymous/ComfyUI#1838]
I'm using the ComfyUI notebook from this repo, using it remotely in Paperspace. About a week or so ago, I've began to notice a weird bug - If I load my workflow by dragging the image into the site, it'll put the wrong positive prompt. Anyone else going through that?

Example of what I mean here:

Image - https://files.catbox.moe/vufmxg.png

EXIF data showing the correct prompt
![Captura de tela de 2023-10-24 16-43-00](https://github.com/comfyanonymous/ComfyUI/assets/70975013/88da3db9-5817-400d-9c6a-821ee0be2dcc)

However, notice the positive prompt once I drag and drop the image into ComfyUI - it's from the previous generated batch:

![Captura de tela de 2023-10-24 16-44-37](https://github.com/comfyanonymous/ComfyUI/assets/70975013/3dc7f163-f76a-4431-a5dc-fb7e969094fa)

All of my images that I've generated with any workflow have this mistake now - I can confirm that the the other fields are correctly pasted in when I drag-and-drop (or load) the image into ComfyUI.

Anyone knows how to fix this? Thanks a lot in advance!

**Comment by kronkinatorix** [comment]
I'm having some similar issues. 
1. Images clearly are being generated with something I haven't prompted them for. E.g. prompting for a racoon will create a dog or a cat or a person. 
2. Changing prompts in a completely seperate model / refiner will change the output of the prompt in another model loaded seperately.
3. Fixed seeds will generate different images. e.g. it will make a bird when loaded, then I'll close and restart and it will make a cat. 
Something ain't right.

**Comment by comfyanonymous** [comment]
https://github.com/comfyanonymous/ComfyUI/commit/40963b5a16f717a636c98dae0055224938852c6a

Should be fixed with this.

**Comment by kronkinatorix** [comment]
I tried loading a workflow I made earlier today with the new update pulled and it generated an image of a dog when the prompt indicated ferret - same image of a dog it had generated before, and did this as I decoded/encoded to a different model, generating one with a "ferret" with the third model. Pretty much what had been happening before. 

In case it was just working out the kinks I tried again with a set seed, and it generated a ferret, then worked on that when I decoded / re-encoded it with a second model, then generated a whole new image when I did it again - same prompt same seed. 

Seems to be consistently generating the same image with the same seed on the first pass now though, besides the weirdness at first. 

Still running into an issue where the refiner doesn't seem to do anything though.
